core/src/consensus/light_client/mod.rs

```diff
@@ -16,13 +16,52 @@
 
 pub mod verification;
 pub use self::verification::{verify_header, verify_state, verify_transaction};
+use ckey::SchnorrSignature;
 pub use ctypes::BlockNumber;
-pub use primitives::H256;
+pub use primitives::{H256, H512};
+use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
 
-pub struct Seal(); // tentative depending on Ed25519 merging
+#[derive(PartialEq, Debug)]
+pub struct Seal(Vec<(usize, SchnorrSignature)>); // (index_in_vset, sign); schnorr scheme
```

Comment:
Could you add the comment above the struct definition? We are not adding comment after the line.